httpcore/_models.py

```diff
@@ -481,3 +482,9 @@ async def aclose(self) -> None:
             )
         if hasattr(self.stream, "aclose"):
             await self.stream.aclose()
+
+
+class ProxyMode(enum.IntFlag):
```

Comment:
```suggestion
class ProxyMode(enum.Enum):
```

Reply:
As I said above, we need to usage as flag for possibility of both selection.